<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1692</width>
    <height>944</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SourceHanSansCN-Regular</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #FFFFFF;</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>1661</width>
     <height>835</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="passwordManager_Page">
    <widget class="QLineEdit" name="originalPassword_lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>35</y>
       <width>460</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 4px;
border: 2px solid #979797;</string>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="placeholderText">
      <string>请输入原密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newPassword_lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>460</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 4px;
border: 2px solid #979797;</string>
     </property>
     <property name="placeholderText">
      <string>请输入新密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirmPassword_lineEdit">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>205</y>
       <width>460</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 4px;
border: 2px solid #979797;</string>
     </property>
     <property name="placeholderText">
      <string>请确认新密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>新 密 码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>133</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>原 密 码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>350</y>
       <width>130</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #0D9DDB;
border-radius: 4px;
color:white;</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="concle_btn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>350</y>
       <width>130</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: white;
border-radius: 4px;
color:#0D9DDB ;
border:2px solid #0D9DDB;</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="hospitalInformation_Page">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>院方名称：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="hospitalName_textEdit">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>60</y>
       <width>1271</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius: 4px;
border: 2px solid #979797;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'黑体'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_btn">
     <property name="geometry">
      <rect>
       <x>1457</x>
       <y>69</y>
       <width>130</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #0D9DDB;
border-radius: 4px;
color:white;</string>
     </property>
     <property name="text">
      <string>编辑</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>155</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>院方地址：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="hospitalAddress_textEdit">
     <property name="geometry">
      <rect>
       <x>161</x>
       <y>145</y>
       <width>1271</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius: 4px;
border: 2px solid #979797;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="hospitalAdressSave_btn">
     <property name="geometry">
      <rect>
       <x>1457</x>
       <y>154</y>
       <width>130</width>
       <height>52</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #0D9DDB;
border-radius: 4px;
color:white;</string>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="productPresentation_Page">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>68</y>
       <width>161</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 140px;
height: 35px;
font-size: 35px;
font-family: SourceHanSansCN-Medium, SourceHanSansCN;
font-weight: 500;
color: #2A2A2A;
line-height: 53px;</string>
     </property>
     <property name="text">
      <string>工作机理</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>113</y>
       <width>647</width>
       <height>225</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 647px;
height: 225px;
font-size: 30px;
font-family:  SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #747474;
line-height: 45px;</string>
     </property>
     <property name="text">
      <string>本设备是通过对踝关节的不同角度不同方向环形主被动运动和直线主动运动，达到改善关节活动范围、提高肌力、缓解肌张力、增加本体感觉输入、提高关节的协调性、灵活性和运动速度，从而提高踝关节的运动功能。</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>841</x>
       <y>68</y>
       <width>105</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 105px;
height: 35px;
font-size: 35px;
font-family: SourceHanSansCN-Medium, SourceHanSansCN;
font-weight: 500;
color: #2A2A2A;
line-height: 53px;</string>
     </property>
     <property name="text">
      <string>禁忌症</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>841</x>
       <y>113</y>
       <width>471</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 702px;
height: 359px;
font-size: 27px;
font-family:  SourceHanSansCN-Regular, SourceHanSansCN;
font-weight: 400;
color: #747474;
line-height: 45px;</string>
     </property>
     <property name="text">
      <string>1、踝关节扭伤急性期；
2、踝关节韧带断裂或骨折手术前；
3、使用部位有皮肤破损或皮肤病者；
4、合并心肝肾或造血系统原发疾病踝关节慢性不稳定、距骨脱位与距骨骨折脱位。</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="system_Page">
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>58</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>语言</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="english_RadioButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>73</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>英文</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>148</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>版本</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="chinese_RadioButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>73</y>
       <width>90</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>中文</string>
     </property>
    </widget>
    <widget class="QLabel" name="version_Label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>V1.0.0.1</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>107</y>
     <width>1642</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">width: 1642px;
height: 2px;
background: #0D9DDB;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>25</y>
     <width>1081</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="hospitalInformation_btn">
    <property name="geometry">
     <rect>
      <x>198</x>
      <y>10</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EFEFEF;
border-radius: 4px;
color:#333333;</string>
    </property>
    <property name="text">
     <string>院方信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="passwordManager_btn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EFEFEF;
border-radius: 4px;
color:#333333;</string>
    </property>
    <property name="text">
     <string>密码管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="productPresentation_btn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #0D9DDB;
font: 20pt &quot;黑体&quot;;
border-radius: 4px;
color:white;</string>
    </property>
    <property name="text">
     <string>产品介绍</string>
    </property>
   </widget>
   <widget class="QPushButton" name="systemSetting_Btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>150</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #EFEFEF;
border-radius: 4px;
color:#333333;</string>
    </property>
    <property name="text">
     <string>系统设置</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
